src/app/cli/src/services/config/models.rs

```diff
@@ -309,6 +315,55 @@ impl Default for JupyterConfig {
 
 ////////////////////////////////////////////////////////////////////////////////////////
 
+#[skip_serializing_none]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "camelCase")]
+#[serde(tag = "kind")]
+pub enum DatabaseConfig {
+    Sqlite(SqliteDatabaseConfig),
+    Remote(RemoteDatabaseConfig),
```

Comment:
ClientServer would be a bit more correct term, but this is minor.

Reply:
I wasn't happy with my naming proposition through, I like yours much better -- renamed